app/api/tax.py

```diff
@@ -57,24 +72,13 @@ def query(self, view_kwargs):
             query_ = query_.join(Event).filter(Event.identifier == event.id)
         return query_
 
-    def before_create_object(self, data, view_kwargs):
-        event = None
-        if view_kwargs.get('event_id'):
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
-        elif view_kwargs.get('event_identifier'):
-            event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'event_identifier')
-        if event:
-            data['event_id'] = event.id
-
     view_kwargs = True
-    decorators = (jwt_required, )
```

Comment:
@shubham-padia we don't want to share any information regarding the organizer or whoever's billing address is being provided. That may or may not be publicly available. So no personal information like that including the company which has registered should be shown according to me.